test-suite/src/data/schema.ts

```diff
@@ -310,7 +310,97 @@ const schema: Schema = {
             ]
           }
-        }
+        },
+        {
+          testName: "Patient sent to A&E",
+          testDescription:
+            "Patient referred to A&E  [e.g. Dx02 - Attend Emergency Treatment Centre within 1 hour]",
+          template: require("./primary-to-a&e-request.xml"),
+          testSpecifications: {
+            [RequestHeaderProps.Header]: [
+              {
+                label: "Url",
+                id: "url",
+                defaultValue: "http://localhost:8081/report",
+                validators: [isUrl(), isReportUrl()],
+              },
+              {
+                label: "Content Type",
+                id: "content-type",
+                defaultValue: "application/xml",
+                validators: [notNull(), minLength(7)],
+              },
+            ],
+            [RequestHeaderProps.Body]: [
+              {
+                label: "ODS Code",
+                id: "ods-code",
+                defaultValue: "EM396",
+                validators: [notNull()],
+              },
+              {
+                label: "DOS Code",
+                id: "dos-code",
+                defaultValue: "26428",
+                validators: [],
+              },
+              {
+                label: "NHS Number",
+                id: "nhs-number",
+                defaultValue: "1112223344",
+                validators: [notNull(), isNumeric(), isLength(10)],
+              },
+              {
+                label: "Recipient Name",
+                id: "recipent-name",
+                defaultValue: "John Stones",
+                validators: [notNull(), isAlpha()],
+              },
+              {
+                label: "Recipient Address Line 1",
+                id: "recipent-street-address-line-1",
+                defaultValue: "99 Made up Address",
+                validators: [notNull()],
+              },
+              {
+                label: "Recipient Address Line 2",
+                id: "recipent-street-address-line-2",
+                defaultValue: "Made up Street",
+                validators: [notNull()],
+              },
+              {
+                label: "Recipient Town",
+                id: "recipent-street-address-town",
+                defaultValue: "Made up Town",
+                validators: [],
+              },
+              {
+                label: "Recipient City",
+                id: "recipent-street-address-city",
+                defaultValue: "Made up City",
+                validators: [],
+              },
+              {
+                label: "Recipient Postcode",
+                id: "recipent-street-address-postcode",
+                defaultValue: "M27 1XR",
+                validators: [notNull()],
+              },
+              {
+                label: "Encounter Start Datetime",
+                id: "nhs111-encounter-date-time-start",
+                defaultValue: "202201061200",
+                validators: [notNull()],
```

Comment:
you can add a date placeholder here using `placeholder: datePlaceholder`. That way, if there's nothing in the input box the user will have some idea of the format that is expected. You could also add the `isNumeric()` validator.